Dockerfile

```diff
@@ -0,0 +1,168 @@
+FROM alpine:3.19.1
+FROM node:18-alpine
+
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+#==================
+# General Packages
+#------------------
+# ca-certificates
+#   SSL client
+# curl
+#   Transfer data from or to a server
+# gnupg
+#   Encryption software. It is needed for nodejs
+# libgconf-2-4 (not available in alpine)
+#   Required package for chrome and chromedriver to run on Linux
+# libqt5webkit5 (not available in alpine)
+#   Web content engine (Fix issue in Android)
+# openjdk-11-jdk
+#   Java
+# sudo
+#   Sudo user
+# tzdata
+#   Timezone
+# unzip
+#   Unzip zip file
+# wget
+#   Network downloader
+# xvfb
+#   X virtual framebuffer
+# zip
+#   Make a zip file
+#==================
+RUN apk update && apk add --no-cache \
+    ca-certificates \
+    curl \
+    gnupg \
+    openjdk11 \
+    sudo \
+    tzdata \
+    unzip \
+    wget \
+    openjdk11 \
+    xvfb-run \
+    bash \
+    rsync \
+    zip \
+    && rm -rf /var/cache/apk/*
+#===============
+# Set JAVA_HOME
+#===============
+
+
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk \
+    PATH=$PATH:$JAVA_HOME/bin
+
+#===============================
+# Set Timezone (UTC as default)
+#===============================
+ENV TZ=UTC
+RUN apk add --no-cache tzdata && \
+    cp /usr/share/zoneinfo/${TZ} /etc/localtime && \
+    echo "${TZ}" > /etc/timezone && \
+    apk del tzdata
+
+
+#===============
+# Create a user
+#===============
+ARG USER_PASS=secret
+RUN addgroup -g 1301 androidusr && \
+    adduser -u 1300 -G androidusr -D -s /bin/sh androidusr && \
+    echo 'androidusr ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
+
+WORKDIR /home/androidusr
+
+#=====================
+# Install Android SDK
```

Comment:
1. collect full list of the executable binaries inside android sdk inside /opt/android
2. inspect appium code to see if any binary used except adb